### PR TITLE
Fix/5082 remove typography prop from ListItemTag component

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 5082 .

#### Changes proposed in this Pull Request:
- Fixed: ListItemTag was using text from react native instead of react native paper
- Removed font prop from ListItemTag
- Removed unused color prop since we have fontColor prop

#### Screenshots / Screen Recording (if applicable)
-
Screenshot 2023-12-27 at 2 27 27 PM

#### To Test:
-

### CODE REVIEW

**Comment by ektaghag-eaton** [comment]
The following screenshots are with all default colors (not use any prop such fontColor, backgroundColor, iconColor)

**Channel Value**

Screenshot 2023-12-27 at 4 57 20 PM

Screenshot 2023-12-27 at 4 57 34 PM

**Empty State**

Screenshot 2023-12-27 at 5 01 25 PM

Screenshot 2023-12-27 at 5 01 15 PM

**Header**
Screenshot 2023-12-27 at 5 03 30 PM

Screenshot 2023-12-27 at 5 03 45 PM

**Hero**
Screenshot 2023-12-27 at 5 04 50 PM

Screenshot 2023-12-27 at 5 04 39 PM

**InfolistItem** (Icon color is not as per design. Issue is already logged here BLUI-5068)
Screenshot 2023-12-27 at 5 05 47 PM
Screenshot 2023-12-27 at 5 05 57 PM

**Mobile Stepper**
Screenshot 2023-12-27 at 5 07 09 PM

Screenshot 2023-12-27 at 5 06 57 PM

**User Menu**
Screenshot 2023-12-27 at 5 08 15 PM
Screenshot 2023-12-27 at 5 08 25 PM

File: docs/ListItemTag.md [hunk 1]
```diff
@@ -24,8 +24,9 @@ import { ListItemTag } from '@brightlayer-ui/react-native-components';
 | --------------- | ----------------------------- | ----------------------------- | -------- | ------------------------- |
 | backgroundColor | Background color of the label | `string`                      | no       | `theme.colors.primary`    |
 | fontColor       | Text color for the label      | `string`                      | no       | Varies based on the theme |
+| fontSize        | Size of the text              | `number`                      | no       | 10                        |
 | label           | The label text                | `string`                      | yes      |                           |
-| theme           | Theme value overrides         | `$DeepPartial<ExtendedTheme>` | no       |                           |
+| theme           | Theme value overrides         | `$DeepPartial<ExtendedTheme>` | no       | `theme.colors.onPrimary`  |
```
Comment (huayunh) [inline docs/ListItemTag.md:29]: why is this the default theme value?

**Review by huayunh — COMMENTED** [review]
(I don't understand the goal of this story; better find a developer to review this)

File: docs/ListItemTag.md (hunk 1 above)
Comment (ektaghag-eaton) [reply docs/ListItemTag.md:29]: These changes were done before our discussion. I have removed and fixed the default theme value [here](https://github.com/etn-ccis/blui-react-native-component-library/pull/482/files#diff-81c387f5a87d21c14fd9236048d5c046b799836a3fe677fa2906d34e4dd6049fR65)

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Pull request merged**